packages/core/src/lib/Slider/Slider.styles.ts

```diff
@@ -0,0 +1,57 @@
+import styled, { css } from 'styled-components';
+import {color, fonts, resetCSS} from '@web3uikit/styles';
+
+// //////////////////
+// Styled Components
+// CSS Props should be sorted alphabetically
+// //////////////////
+
+
+const sliderDiv = styled.div`
+    
```

Comment:
It would be nice if you could change the naming convention.. and follow same as rest of the components
for eg - DivStyled, LabelStyled, InputStyled.....

Reply:
will do